### PR TITLE
[WIP] Seer v1. Breaking changes to instrument struct, notate not currently …

### PR DESCRIPTION
**Pull request by troy-f**
…functional.

### CODE REVIEW

**Comment by troy-f** [comment]
this is ready for review now

File: BardMusicPlayer.Notate/Processor/AutoToneProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+            if (!Config.AutoToneInstrumentGroup.Instrument1.Equals(Instrument.None)) notes.AddRange(instrumentGroup1.Result);
+            if (!Config.AutoToneInstrumentGroup.Instrument2.Equals(Instrument.None)) notes.AddRange(instrumentGroup2.Result);
+            if (!Config.AutoToneInstrumentGroup.Instrument3.Equals(Instrument.None)) notes.AddRange(instrumentGroup3.Result);
+            
+            var trackChunk = TimedObjectUtilities.ToTrackChunk(notes);
+
+            var playerNotesDictionary = await trackChunk.GetPlayerNoteDictionary(Config.PlayerCount, OctaveRange.C3toC6.LowerNote, OctaveRange.C3toC6.UpperNote);
+
+            var concurrentPlayerTrackDictionary = new ConcurrentDictionary<long, TrackChunk>();
```
Comment (trotlinebeercan) [inline BardMusicPlayer.Notate/Processor/AutoToneProcessor.cs:71]: this should be preallocated for speed gains / step performance when populated in the parallel loop below

**Comment by trotlinebeercan** [comment]
PR looks good to me, after discussing a few changes in Discord.

**Review by trotlinebeercan — DISMISSED** [review]
Approving PR.

**Pull request merged**